web/themes/custom/fmcg_material/sass/styles.scss

```diff
@@ -2273,3 +2210,36 @@ form#views-exposed-form-job-board-page {
     border-radius: 2px;
   }
 }
+
+@media (min-width: $lg) {
+  .navbar-hamburger-menu {
+    display: none !important;
```

Comment:
Can we avoid important? If not fine. 

I think in this case, we can actually just add the "invisible-lg" class to the block.